### ISSUE TITLE
Example how to use Powershell Deployment

### ISSUE THREAD

**Issue by taliesins** [sponte/Powershell.Deployment#28]
Would be helpful to document how to use this :>

**Comment by sponte** [comment]
@taliesins Agree, but I won't have time to work on this unfortunately. There are some instructions on the homepage. Hopefully whoever wants to use this will have enough technical knowledge to understand on high level how it works

**Comment by kemmis** [comment]
+1
